index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portifolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <section class="section-navbar">
+      <nav>
+        
+        <div class="toolbar-txt">My Logo</div>
+
+          <a href="#" class="toggle-button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </a>
+
+      </nav>
+
+    </section>
```

Comment:
- [ ] The toolbar is usually wrapped with the `header` tag. The header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Kindly change the `section` tag to a `header` tag.
 - [ ] The logo is usually a link to the home page on most web pages. For a better user experience, kindly change the `div` tag to an `a` tag.
 - [OPTIONAL] The hamburger button will be a button that opens the mobile menu. In future lessons, you will build this mobile menu and the wrapper of the hamburger button would be better to be the `button` tag.